app-i18n/man-pages-zh_CN/man-pages-zh_CN-1.6.3.2.ebuild

```diff
@@ -0,0 +1,38 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+DESCRIPTION="A somewhat comprehensive collection of Chinese Linux man pages"
+HOMEPAGE="https://github.com/man-pages-zh/manpages-zh"
+MY_PN="manpages-zh"
+MY_P="${MY_PN}-${PV}"
+SRC_URI="https://github.com/${PN}/${MY_PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+S="${WORKDIR}/${MY_P}"
+
+LICENSE="FDL-1.2"
+SLOT="0"
+KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~hppa ~ia64 ~m68k ~mips ~ppc ~ppc64 ~s390 ~sh ~sparc ~x86"
```

Comment:
```
RepoMan scours the neighborhood...
  KEYWORDS.dropped              1
   app-i18n/man-pages-zh_CN/man-pages-zh_CN-1.6.3.2.ebuild: amd64-linux x86-linux
```

Is there a reason why you've removed `~amd64-linux ~x86-linux `keywords?

Reply:
I found there's no ~amd64-linux ~x86-linux in other man-pages-xxx packages, so I removed these keywords.

Reply:
Yeah, Prefix people are not really consistent or predictable in adding those keywords, and they are combining some packages from overlay which makes it even less predictable. However, please keep them unless `repoman full -d` requires you to remove them (and it won't because prefix is marked as experimental).